etor/src/main/java/gov/hhs/cdc/trustedintermediary/etor/ruleengine/transformation/custom/UpdateReceivingApplicationNamespace.java

```diff
@@ -13,11 +13,19 @@
 public class UpdateReceivingApplicationNamespace implements CustomFhirTransformation {
 
     @Override
-    public void transform(FhirResource<?> resource, Map<String, String> args) {
+    public void transform(FhirResource<?> resource, Map<String, Object> args) {
         Bundle bundle = (Bundle) resource.getUnderlyingResource();
+        String name = (args.get("name") instanceof String ? (String) args.get("name") : null);
         var receivingApplication = HapiHelper.getMSH5MessageDestinationComponent(bundle);
+        if (name == null) {
```

Comment:
Same as my comment above. I don't think we want to return null when an expected value in the args is not there or doesn't have the right type

Reply:
same